docs/v1/p4runtime-id-notes.md

```diff
@@ -0,0 +1,487 @@
+# P4Runtime numeric id notes
+
+There are many kinds of numeric identifiers used in the P4Runtime API
+specification.  While they are all described in the specification and
+accompanying Protobuf `.proto` files, this article attempts to collect
+in one place notes on those identifiers, including:
+
++ Which software entity (or person) chooses their values
++ When the values are chosen
++ What is their scope in "space" and time in which they must be unique
+
+This document was written while referring to the P4Runtime API
+specification as of this commit of the Github repository
+https://github.com/p4lang/p4runtime
+
+```
+commit c9cd4af17c2d562d6e4ec760237f16fea9558fcb
+Author: Chris Sommers <31145757+chrispsommers@users.noreply.github.com>
+Date:   Fri Mar 27 13:43:31 2020 -0700
+```
+
+This is after version 1.1.0 of the specification was released, and it
+is expected that version 1.2.0 will be released soon after this time,
+with very few (if any) changes from the version above.
```

Comment:
@jafingerhut please remind me to update this text when I do the v1.2.0 release

Reply:
Sure, I can do that, and/or create a PR to update it myself.